### PR TITLE
Refs #30426 -- Changed default SECURE_CONTENT_TYPE_NOSNIFF to True.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

**Comment by claudep** [comment]
Removed the SECURE_BROWSER_XSS_FILTER part as per https://code.djangoproject.com/ticket/30680

File: docs/releases/3.0.txt [hunk 1]
```diff
@@ -506,6 +506,8 @@ Miscellaneous
   :class:`~django.db.models.OneToOneField` ``'_id'`` attribute now unsets the
   corresponding field. Accessing the field afterwards will result in a query.
 
+* :setting:`SECURE_CONTENT_TYPE_NOSNIFF` setting defaults to True.
```
Comment (adamchainz) [inline docs/releases/3.0.txt]: I think you need double backticks around `True`

File: docs/releases/3.0.txt [hunk 2]
```diff
@@ -519,6 +519,12 @@ Miscellaneous
   field names contains an asterisk, then the ``Vary`` header will consist of a
   single asterisk ``'*'``.
 
+* :setting:`SECURE_CONTENT_TYPE_NOSNIFF` setting now defaults to ``True``. With
+  the enabled :setting:`SECURE_CONTENT_TYPE_NOSNIFF`, the
+  :class:`~django.middleware.security.SecurityMiddleware` sets the
+  :ref:`x-content-type-options` header on all responses that do not already
+  have it.
```
Comment (felixxm) [inline docs/releases/3.0.txt]: I added a line of explanation to the release notes.

**Comment by felixxm** [comment]
@claudep Thanks!

**Pull request merged**